### ISSUE TITLE
text written in vim is invisible to reddit

### ISSUE THREAD

**Issue by mirsella** [glacambre/firenvim#625]
Hello

i'm sorry if you don't take issue about specific site, i didn't found any info about this.

the problem i'm facing is that only in commenting mode (RPAN live chat too) the text i wrote in nvim after i :wq become invisible to reddit. i can't post because the text field is empty for reddit.
i can only post after i typed some character without nvim launched. 
before i clic the post button the text written in nvim is here, but after posted it diseapper.
i've not seen how to got the log of firenvim.

i'm not english, my explanation is really bad. 
that why i recored a little clip of the problem (20 seconds) : https://youtu.be/petMN_2-INE

also while i'm here i can't paste anything to chromium when i got the extension, tell me if you me to create another issue. while writing this i got a error related to clipboard too : https://i.imgur.com/mt1b7HZ.png
i got set clipboard=unnamedplus in my vimrc.

my firenvim config in vimrc :
```
let g:firenvim_config = { 
  \ 'globalSettings': {
  \  },
  \ 'localSettings': {
  \ },
  \ }
let fc = g:firenvim_config['localSettings']
let fc['https?://discord\.com'] = { 'takeover': 'never', 'priority': 1 }
let fc['.*'] = { 'takeover': 'once' }
  ```

**Comment by glacambre** [comment]
Hi :),

I can't reproduce your problem. I see that you have a lot of browser extensions - could you try to use a fresh chrome profile with only Firenvim and see if the problem still happens?

Also I don't think Firenvim is responsible for your clipboard issue - this is entirely handled by neovim. Try running `:checkhealth` to see if neovim has anything to say about your clipboard.

**Comment by mirsella** [comment]
i've seen that the problem of text field not registering the text written in nvim is not only on reddit but on twitter too. didn't tried a lot of other site. github and discord work.
i tried with a clean chromium and a clean chrome instance, both with a empty vimrc (just the plugins sourced) and the problem is still here. i don't understand this, a full clean setup should work. i'm currently installing a VM to test it here.

for the clipboard problem, it's totally my bad i'm a dumbass, i've set CTRL+V to enable/disable firenvim as a shortcut. it's fixed now.

**Comment by glacambre** [comment]
Twitter not working is a known issue (https://github.com/glacambre/firenvim/issues/322) - but reddit should work. I'm not sure what could be going on here.

**Comment by mirsella** [comment]
the issue of OP doesn't seems to be the same as me, but on this issue i found this comment : https://github.com/glacambre/firenvim/issues/322#issuecomment-579156992 which seems to be exactly my problem. 
i also tested on facebook and the same problem occur.

**Comment by mirsella** [comment]
the issue of OP doesn't seems to be the same as me, but on this issue i found this comment : https://github.com/glacambre/firenvim/issues/322#issuecomment-579156992 which seems to be exactly my problem. 
i also tested on facebook and the same problem occur.

tell me who i can help and send more information.

**Comment by mirsella** [comment]
just tried with a brand new virtual machine. both with chromium and firefox, and a clean nvimrc. still doesn't work sadly.

**Comment by glacambre** [comment]
@mirsella I had to create a new account and now I can reproduce. The issue is that the markdown editor uses React, which doesn't react to synthetic events. I'll try to find a workaround.

In the meantime, you can use markdown mode to use Firenvim on reddit.

**Comment by mirsella** [comment]
thanks for looking into it.
Reddit have a switch to enter markdown mode, but for me it does the same for Twitter and Facebook (I didn't tried other one)

**Comment by mirsella** [comment]
yup, just tried, work on reddit with markdown mode (you can set it by default) but not working on twitter
